### DISCUSSION TITLE
Jupyter notebook as a README

### DISCUSSION THREAD

**Discussion by donno2048** [community/community#5233] [Other Feature Feedback, Questions, & Ideas]
It would be great to be able to use a _README.ipynb_ and there are over thirty people on isaacs/github#1425 already asking for it

**Comment by michaelmior** [comment]
While it's obviously not the same thing, you could set up a GitHub action to convert README.ipynb to README.md on every commit with nbconvert.

**Comment by rezaarefi** [comment]
09370803033
Gmail: arefi4264@gmail.com

**Comment by rezaarefi** [comment]
https://senatormobile.wordpress.com
